layouts/header.html

```diff
@@ -1,13 +1,35 @@
-<div id="header-wrapper">
-  <div id="header">
-    <div>
+<div id="header">
+
+  <div id="top-header">
+    <div class="header-wrapper">
+
       <a class="logo" href="/">Cerner Corporation</a>
-      <ul class="nav">
+
+      <nav id="navigation">
+        <ul>
+          <li><a target="_blank" href="http://code.cerner.com/learn">Learn</a></li>
+          <li><a target="_blank" href="http://code.cerner.com/browse">Browse</a></li>
+          <li><a target="_blank" href="http://code.cerner.com/build">Build</a></li>
+          <li><a target="_blank" href="http://code.cerner.com/engage">Engage</a></li>
+        </ul>  
+      </nav>
+
+      <a id="start-coding" target="_blank" href="http://fhir.cerner.com/may2015">Start Coding</a>
```

Comment:
Do we want this link to be here? It works on code because it links to another site. This ends up just linking back to the same website you're on currently.